mava/systems/README.md

```diff
@@ -6,44 +6,17 @@ training. This allows for easy scaling of computational resources by changing on
 
 Below we list the different systems in Mava based on the action spaces they use. More systems will be added in future Mava updates.
 
-### Continuous control
-
-The following systems focus on this
-setting:
-
-System                                                                | Paper                    | Code
--------------------------------------------------------------------- | :----------------------: | :--:
-Multi-Agent Deep Deterministic Policy Gradient (MADDPG)             | [Lowe et al., 2017]   | [![TF][TF Logo]][MADDPG_TF2]
-Multi-Agent Distributed Distributional DDPG (MAD4PG)    | [Barth-Maron et al., 2018] | [![TF][TF Logo]][MAD4PG_TF2]
-
-### Discrete control
-We also include a number of systems built with discrete action-spaces in mind listed below:
-
-System                                                    | Paper                    | Code
--------------------------------------------------------- | :----------------------: | :--:
-Deep Q-Networks (DQN)                                    | [Horgan et al., 2018]      | [![TF][TF Logo]][DQN_TF2]
-VDN                                                     | [Sunehag et al., 2017]      | [![TF][TF Logo]][Value Decomposition_TF2]
-QMIX                                                     | [Rashid et al., 2018]      | [![TF][TF Logo]][Value Decomposition_TF2]
+For our TF2-based systems (maddpg, madqn, vdn and qmix), please install [`v0.1.3`](https://github.com/instadeepai/Mava/releases/tag/0.1.3) of Mava (e.g. `pip install id-mava==0.1.3`). We will no longer be supporting these systems as we have moved to JAX-based systems.
```

Comment:
For my other comment on "Jax-based", I guess it makes sense to keep the distinction in places like this, however, we will probably at some point remove this completely, i.e. not mention TF2.

Reply:
Agreed.